src/lib/prepareOas.ts

```diff
@@ -40,7 +40,17 @@ export default async function prepareOas(path: string, command: 'openapi' | 'val
 
     const fileFindingSpinner = ora({ text: 'Attempting to locate API definitions...', ...oraOptions() }).start();
```

Comment:
this one's on me lol but mind changing this too? in hindsight I hate this original language I wrote lol

```suggestion
    const fileFindingSpinner = ora({ text: 'Looking for API definitions...', ...oraOptions() }).start();
```